### PR TITLE
Fix settings buttons animation inconsistent

### PR DESCRIPTION
**Pull request by Rahmon**
## Description
Same visual button but inconsistent behaviour #3746 

## How Has This Been Tested?
This has been tested with "npm test", "npm run test-e2e" and manually on Chrome and Firefox.

## Types of changes
Bug fix

## Checklist:
- [X] My code is tested.
- [X] My code follows the WordPress code style.
- [ ] My code follows has proper inline documentation.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ const config = {
 		packageNames.reduce( ( memo, packageName ) => {
 			memo[ packageName ] = `./node_modules/@wordpress/${ packageName }`;
 			return memo;
-		}, {} ),
```
Comment (youknowriad) [inline webpack.config.js]: Maybe we could avoid this change? Trailing commas are recommended in Gutenberg

**Review by youknowriad — APPROVED** [review]
LGTM 👍

File: webpack.config.js (hunk 1 above)
Comment (Rahmon) [reply webpack.config.js]: When a tried run "npm run dev" I got unexpected token ')' in line 84.

Comment (youknowriad) [reply webpack.config.js]: It looks like you are using `node 6` which had a bug and the same thing was merged into master yesterday to fix it. Might be better to switch to `node 8`anyway as the recommended version.

#3757

**Pull request merged**